src/js/components/Skeleton/__tests__/Skeleton-test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import 'jest-styled-components';
+
+import { Skeleton } from '..';
+import { Grommet } from '../../Grommet';
+import { Box } from '../../Box';
+import { Text } from '../../Text';
+import { Heading } from '../../Heading';
+import { Paragraph } from '../../Paragraph';
+import { Button } from '../../Button';
+
+describe('Skeleton', () => {
+  test('renders', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Skeleton />
+      </Grommet>,
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('Box skeleton loading', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Box skeleton>
+          <Heading>Heading</Heading>
+          <Text>Text</Text>
+          <Paragraph>Paragraph</Paragraph>
+          <Button label="Button" onClick={() => {}} />
+        </Box>
+      </Grommet>,
+    );
+    expect(screen.queryByText('Heading')).toBeNull();
+    expect(screen.queryByText('Text')).toBeNull();
+    expect(screen.queryByText('Paragraph')).toBeNull();
+    expect(screen.queryByText('Button')).toBeNull();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('Box skeleton sizes loading', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Box skeleton>
+          <Heading level={2} size="small">
+            Heading
+          </Heading>
+          <Text size="xsmall">Text</Text>
+          <Paragraph size="large">Paragraph</Paragraph>
+          <Button size="large" label="Button" onClick={() => {}} />
+        </Box>
+      </Grommet>,
+    );
+
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Should we add expects here that:

`expect(screen.queryByText('Heading')).toBeNull();`